scripts/automated_ingestion/eessitarball.py

```diff
@@ -393,7 +396,11 @@ def make_approval_request(self):
             )
             pr_title = '[%s] Ingest %s' % (self.cvmfs_repo, filename)
             if self.sig_verified:
-                pr_body += "\n\n:heavy_check_mark: :closed_lock_with_key: The signature of this tarball has been successfully verified."
+                pr_body += "\n\n:heavy_check_mark: :closed_lock_with_key: The signature of this tarball has been successfully verified:\n"
+                for path, meta in self.signatures.items():
+                    identity = meta.get("identity", "unknown")
+                    namespace = meta.get("namespace", "unknown")
+                    pr_body += f"- `{path}`: identity=`{identity}`, namespace=`{namespace}`\n"
```

Comment:
Make it a little easier to skim quickly
```suggestion
                    pr_body += f"- `{path}`\n  - identity=`{identity}`, namespace=`{namespace}`\n"
```